codegeneration/codegen.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+rm -rf ./build ../src/main/java/com/adobe/target/delivery/v1
+mkdir ../src/main/java/com/adobe/target/delivery/v1
+git clone git@github.com:dcottingham/openapi-generator.git
+cd openapi-generator
+git fetch --tags
+git checkout tags/v5.2.0_custom
+mvn clean install package -Dmaven.javadoc.skip=true -Dmaven.test.skip=true
```

Comment:
Should we add that maven is a required dependency? Can we try to do this with `gradle`?

Reply:
BTW I have tried this instead and it worked:
```
./mvnw clean install package -Dmaven.javadoc.skip=true -Dmaven.test.skip=true
```